src/main/resources/l10n/JabRef_zh_CN.properties

```diff
@@ -2515,7 +2525,20 @@ This\ could\ indicate\ that\ JabRef\ did\ not\ shut\ down\ cleanly\ last\ time\
 
 Use\ the\ field\ FJournal\ to\ store\ the\ full\ journal\ name\ for\ (un)abbreviations\ in\ the\ entry=使用FJournal字段记录Journal全称
 
+Library\ to\ import\ into=Library to import into
 
+Multiline=Multiline
 
+Unable\ to\ open\ linked\ eprint.\ Please\ set\ the\ eprinttype\ field=Unable to open linked eprint. Please set the eprinttype field
```

Comment:
This does not seem to be a chinese translation...

Reply:
Comment that on crowdin at the string

Reply:
Removed translation on Crowdin.

Reply:
Maybe also add a hint that Jabref will auto use English if the string is not translated